### DISCUSSION TITLE
What is your favourite animal?

### DISCUSSION THREAD

**Discussion by taiyo-sun25** [NAIST-SE/AttractingContributors#1049] [General]
https://github.com/taiyo-sun25/AttractingContributors-animals

**Comment by mariawotters** [comment]
I adore cats! But I’ve never had a cat of my own. I think I’m ready to get one now. I’ve already put together a first-aid kit from [Canadian Pharmacy](https://www.canadapharmacy.com/) , and I’ve also ordered all the essentials for a cat to live comfortably at my place. Now, all that’s left is to find a fluffy friend! I was thinking of visiting a shelter to choose one. This way, I can help at least one cat in need.
